src/coreclr/interpreter/interpreter.h

```diff
@@ -15,6 +15,12 @@
 #ifdef HOST_WINDOWS
 #include <malloc.h>
 #endif
+
+// HACK: debugreturn.h breaks constexpr
+#if defined(debug_instrumented_return) || defined(_DEBUGRETURN_H_)
```

Comment:
Just curious - I wonder what pulls in the debugreturn.h

Reply:
I believe it's coming from PCH, i'll try to find out

Reply:
According to the compiler, it's gcinfotypes.h pulling it in. A few different places pull in gcinfotypes, like eetwain.h and gcinfoencoder.h.

Reply:
Before gcinfotypes.h defines it, it comes from debugreturn.h but it's not telling me what included it... contract.h does, so maybe it's that?

Reply:
Sadly /showIncludes doesn't seem to work in our build environment, perhaps cmake or ninja are eating the output...

Reply:
Thank you for checking!